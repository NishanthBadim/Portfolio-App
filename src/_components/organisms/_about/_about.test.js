import React from 'react'
import { create } from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect"
import About from './_about'

jest.mock('./../../molecules/_experience/_experience', () => ({
    __esModule: true,
    default: () => <div>I am mocked</div>
}));

jest.mock('./../../molecules/_education/_education', () => ({
    __esModule: true,
    default: () => <div>I am mocked</div>
}));

jest.mock('./../../molecules/_skills/_skills', () => ({
    __esModule: true,
    default: () => <div>I am mocked</div>
}));

describe("About Component Testing", () => {
    test("Match the Snapshot", () => {
        const input = create(<About />)
        expect(input.toJSON()).toMatchSnapshot();
    })
})