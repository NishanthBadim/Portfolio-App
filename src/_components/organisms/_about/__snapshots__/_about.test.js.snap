// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component Testing Match the Snapshot 1`] = `
<div>
  <div
    className="mdl-grid"
  >
    <div
      className="mdl-cell mdl-cell--5-col"
    >
      <div
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        <img
          alt="avatar"
          className="makeStyles-root-2"
          src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRDXh6VSPmGsJQ8SOQP5vdAXibWEJSd1ZXUZA&usqp=CAU"
        />
      </div>
      <div>
        <h2
          style={
            Object {
              "paddingTop": "1.5em",
            }
          }
        >
          Nishanth Badim
        </h2>
        <h4
          style={
            Object {
              "color": "grey",
            }
          }
        >
          Web Developer
        </h4>
        <hr
          style={
            Object {
              "borderTop": "3px solid #833fb2",
              "width": "60%",
            }
          }
        />
        <p>
          My personal philosophy when it comes to learning is closely related to several theories of motivation. I look things happening around me as simple as possible and postively and . I love designing websites and I wish my passion for web desinging continues.
        </p>
        <hr
          style={
            Object {
              "borderTop": "3px solid #833fb2",
              "width": "60%",
            }
          }
        />
        <h5>
          Address
        </h5>
        <p>
          Street No: 25, Hyderabad 500 062
        </p>
        <h5>
          Phone
        </h5>
        <p>
          (123) 456-7890
        </p>
        <h5>
          Email
        </h5>
        <p>
          nishanth.badim@zemosolabs.com
        </p>
        <hr
          style={
            Object {
              "borderTop": "3px solid #833fb2",
              "width": "50%",
            }
          }
        />
      </div>
    </div>
    <div
      className="mdl-cell mdl-cell--7-col makeStyles-root-1"
    >
      <h2>
        Education
      </h2>
      <div>
        I am mocked
      </div>
      <hr
        style={
          Object {
            "borderTop": "3px solid #e22947",
          }
        }
      />
      <h2>
        Experience
      </h2>
      <div>
        I am mocked
      </div>
      <div>
        I am mocked
      </div>
      <hr
        style={
          Object {
            "borderTop": "3px solid #e22947",
          }
        }
      />
      <h2>
        Skills
      </h2>
      <div>
        I am mocked
      </div>
      <div>
        I am mocked
      </div>
      <div>
        I am mocked
      </div>
      <div>
        I am mocked
      </div>
      <div>
        I am mocked
      </div>
    </div>
  </div>
</div>
`;
